lib/ruby_lsp/setup_bundler.rb

```diff
@@ -202,7 +202,7 @@ def run_bundle_install(bundle_gemfile = @gemfile)
       end
 
       # If `ruby-lsp` and `debug` (and potentially `ruby-lsp-rails`) are already in the Gemfile, then we shouldn't try
-      # to upgrade them or else we'll produce undesired source control changes. If the custom bundle was just created
+      # to upgrade them or else we'll produce undesired source control changes. If the composed bundle was just created
       # and any of `ruby-lsp`, `ruby-lsp-rails` or `debug` weren't a part of the Gemfile, then we need to run `bundle
       # install` for the first time to generate the Gemfile.lock with them included or else Bundler will complain that
       # they're missing. We can only update if the custom `.ruby-lsp/Gemfile.lock` already exists and includes all gems
```

Comment:
```suggestion
      # they're missing. We can only update if the composed `.ruby-lsp/Gemfile.lock` already exists and includes all gems
```

We have references to a composed lockfile. I am assuming this is it. Should we also refer to it here as `composed`?